proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,91 @@
+# cert-manager Incubation Proposal
+
+cert-manager joined the CNCF as a sandbox project just over a year ago, and we firmly believe that the project is ready to take the next step and graduate to the Incubation stage.
+
+cert-manager provides reliable, automated Kubernetes-native management of X.509 certificates. It reduces the risk of certificate-related outages and enables PKI at scale, and is treated by many teams as de-facto part of the core components of a Kubernetes or OpenShift deployment.
+
+The project comprises cert-manager itself - which adds Kubernetes Custom Resource Definitions (CRDs) for certificate-related concepts, and provides controllers to manage them - along with a selection of subprojects which focus on solving other issues relating to certificate management.
+
+The main cert-manager repo has 8.4k stars on GitHub and hundreds of projects downstream which depend on cert-manager code in some way. There are thousands of companies which have contributed in some way and cert-manager's use and prominence remains strong.
+
+## Incubation Criteria Checklist
+
+Based on the [graduation criteria](https://github.com/cncf/toc/blob/da2c6f1a14ea9592ca0d1794da376e0c874b5a6f/process/graduation_criteria.adoc#incubating-stage), cert-manager seems to clearly tick the boxes required for incubation:
+
+### Production Use
```

Comment:
You could point to https://github.com/cert-manager/cert-manager/blob/master/USERS.md, there's not many listed there, but there are 3 non-Jetstack companies listed.

There is this presentation from Walmart where they talk about using cert-manager: https://youtu.be/sfPFrvDvdlk?t=709

I think these are worth including as well, as they are pointing to production use by companies, whereas most of the things listed here are inferring production use from the fact that companies are contributing.